clients/repo_client.go

```diff
@@ -18,6 +18,7 @@ import (
 	"fmt"
```

Comment:
ignore this file, it will disappear once previous https://github.com/ossf/scorecard/pull/712 and https://github.com/ossf/scorecard/pull/718 are merged